### PR TITLE
Adding logging config and update pylintrc to use fstring for logging SCRUM-13

### PR DESCRIPTION
**Pull request by debuggingMyBrain**
https://gamepals.atlassian.net/browse/SCRUM-13

### CODE REVIEW

File: backend/settings.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+def init_logging():
+    logging_format = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+    log_filename = "/var/log/scrumsistant/scrum.log"
+    if platform.system() == "Windows":
+        # For us ghouls that are running on Windows, won't have a /var, so log locally
+        log_filename = "scrum.log"
+    # I would imagine that in production that we would just have a long running service that would log over
+    # a long period, so we might have a size determined rotation, but during development I find it helpful
+    # to restart the logs every run, seeing as different runs are common
```
Comment (DannyPinghero) [inline backend/settings.py:14]: I'm not a fan of this decision - I might want to change a few things and see several runs; the startup log and/or using tail and [enter] is enough to help disambiguate. On your local you can clear the log files, on prod log rotation is a well understood external tool

File: backend/settings.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import logging
+import logging.handlers
+import platform
+
+
+def init_logging():
+    logging_format = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+    log_filename = "/var/log/scrumsistant/scrum.log"
+    if platform.system() == "Windows":
```
Comment (DannyPinghero) [inline backend/settings.py:9]: Good solution

File: backend/settings.py (hunk 1 above)
Comment (debuggingMyBrain) [reply backend/settings.py:14]: So no RotatingLogHandler at all, or just make it size based?

Comment (DannyPinghero) [reply backend/settings.py:14]: Imo none at all but what do you think

Comment (DannyPinghero) [reply backend/settings.py:14]: On CIQ prod we use 
https://linux.die.net/man/8/logrotate

**Pull request merged**